build.sbt

```diff
@@ -775,6 +779,52 @@ lazy val zioJson: ProjectMatrix = (projectMatrix in file("json/zio"))
   )
   .dependsOn(core)
 
+//grpc
+lazy val protobuf: ProjectMatrix = (projectMatrix in file("grpc/protobuf"))
+  .settings(commonSettings)
+  // .enablePlugins(AkkaGrpcPlugin)
+  .settings(
+    name := "tapir-grpc-protobuf",
+    libraryDependencies ++= Seq(
+      "com.typesafe.akka" %% "akka-discovery" % "2.6.19",
+      "ch.megard" %% "akka-http-cors" % "0.4.2"
```

Comment:
we need discovery and cors for protobuf?

Reply:
Cors was redundant, but discovery we need to import explicitly because we use currently Akka `2.6.20`, but `akka-grpc-runtime` brings discovery module `2.6.19`, so the compilation fails with 

```
 Exception in thread "main" java.lang.IllegalStateException: You are using version 2.6.20 of Akka, but it appears you (perhaps indirectly) also depend on older versions of related artifacts. You can solve this by adding an explicit dependency on version 2.6.20 of the [akka-discovery] artifacts to your project. Here's a complete collection of detected artifacts: (2.6.19, [akka-discovery]), (2.6.20, [akka-actor, akka-protobuf-v3, akka-slf4j, akka-stream])
```